index.html

```diff
@@ -93,40 +95,388 @@ <h2 class="box-header">Lorem Ipsum Generator</h2>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+          </ul>
+        </div>
+        <div class= "guide-title">
+          <h5>New Front-End</h5>
+          <ul class="guide-ulist">
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+          </ul>
+        </div>
+        <div class= "guide-title">
+          <h5>New Front-End</h5>
+          <ul class="guide-ulist">
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+          </ul>
+        </div>
+        <div class= "guide-title">
+          <h5>New Front-End</h5>
+          <ul class="guide-ulist">
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+            <li>CSS Grid Layout</li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="smashing-newsletter">
+        <h4>Smashing Newsletter</h4>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          Sagittis vitae et leo duis ut diam quam nulla porttitor.
+        </p>
+        <span class="image-banner"></span>
+        <button>Subscribe</button>
+      </section>
+      
+      <section>
+        <h6 class="display"> Heading </h6>
+        <div class="floating-link">
+          <span>More Links on Twitter <i class="fa fa-arrow-right"></i></span>
+        </div>
+      </section>
     </main>
+
+    <footer class="footer">
+      <div>
+        <h4> BROWSE ALL SMASHING MAGAZINE TOPICS </h4>
+      
+        <p>
+          A cras semper auctor neque vitae tempus quam. 
+          Tellus molestie nunc non blandit massa enim. 
+          Tincidunt arcu non sodales neque sodales ut etiam sit. 
+          Sit amet venenatis urna cursus eget nunc scelerisque. 
+          Massa ultricies mi quis hendrerit dolor. Morbi tincidunt augue interdum velit. 
+          Nibh mauris cursus mattis molestie a iaculis at erat pellentesque.
+        </p>
+
+        <p>
+          Plugins React Responsive Web Design Service Workers Sketch Typography UIUsabilityUser 
+          Experience Wall papersWeb Design WordPress Workflow
+        </p>
+      </div>
+    </footer>
```

Comment:
- The footer should contain centered links. Just change the content to be links as the original page.